consensus/miner_signalled_consensus.md

```diff
@@ -0,0 +1,200 @@
+# Miner Signalled Consensus Upgrade
+
+## Simple Summary
+
+This document describes the proposal of how nodes in the peer-to-peer network
+may activate a new hard-forking consensus protocol based on miner signalling.
+
+## Motivation
+
+Activating a hard-forking new consensus protocol based on miner signalling
+prevents diverging the block chain in case a large fraction of the miners have
+not upgraded their nodes to a version able to follow the new consensus protocol.
+
+## Specification
+
+### Overview
+
+On a node configured for a miner-signalled consensus upgrade,
```

Comment:
Nodes can be configured to handle a specific hard-fork as a reaction on signals from other nodes?

Reply:
It means that the fork signalling needs to be fork aware. It's showed in https://github.com/aeternity/protocol/pull/385#discussion_r325211457

So if the protocol upgrade happens on the current chain, it doesn't necessarily happen on all chain forks.